### DISCUSSION TITLE
Is it possible to open multiple charts from one code?

### DISCUSSION THREAD

**Discussion by PythonAI1234** [highfestiva/finplot#510] [Q&A]
I am using jupyter notebook to test out different strategies and I have a lot of slightly nuanced calculation I would like to juxtapose and see how well it works on different charts.
But now I need to close the previous chart window to run the program again as I use fplt.show(). If I want to compare two I would have to copy the whole notebook.
As the chart is static I feel like there must be a way for it to create a chart that stays without keeping the program running.
Is there anyway to do this?

**Comment by highfestiva** [comment]
Nope, it's an application when you make it one. If you want to compare two calculations, you have to add them to the same chart right away. Comparing two or more strategies can be tricky for several reasons, as determining the quality depends on profit and Sharpe ratio, preferably over several time frames. I suggest you optimize your flow for backtesting, and adapt the tech accordingly. Perhaps finplot is the right choice for your flow, perhaps not.
